account_bank_statement_import_1c/account_bank_statement_import_1c.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding:utf-8 -*-
+"""1C Bank import"""
+from openerp import api, models
+from openerp.tools.translate import _
+import re
+from datetime import datetime as dt
+import os
+
+
+class AccountBankStatementImport(models.TransientModel):
+    _inherit = 'account.bank.statement.import'
+    normalHead = False
+
+    @api.model
+    def _parse_file(self, data_file):
+        result = {'errors': [], 'general': {}, 'remain': {}, 'filters': {},
+                  'documents': []}
+        self.normalHead = data_file.startswith('1CClientBankExchange')
+        if not self.normalHead:
+            return super(AccountBankStatementImport, self)._parse_file(
+                data_file)
+        # parse general info (key=value)
+        data_file = data_file.decode('cp1251').encode('utf-8')
+        for key in ['ВерсияФормата', 'Кодировка', 'Отправитель',
+                    'Получатель', 'ДатаСоздания', 'ВремяСоздания']:
+            st = re.search(key + '=(.*)\r', data_file)
+            if st:
+                result['general'][key] = st.group(1)
+        # normalize
+        data_and_action = [
+            ('ДатаСоздания', lambda y: dt.date(dt.strptime(y, '%d.%m.%Y'))),
+            ('ВремяСоздания', lambda y: dt.date(dt.strptime(y, '%d.%m.%Y'))),
+        ]
+        for x in data_and_action:
+            try:
+                result['general'][x[0]] = x[1](result['general'][x[0]])
+            except AttributeError:
+                pass
+        # parse remain
+        # r prefix makes this string as raw string without \like escapes.
+        regexp_acc = r'СекцияРасчСчет([\s\S]*?)\sКонецРасчСчет'
+        for match in re.findall(regexp_acc, data_file):
+            for match_child in re.findall('(.*)=(.*)\r', match):
+                result['remain'][match_child[0]] = match_child[1]
+        # normalize
+        data_and_action = [
+            ('ДатаНачала', lambda y: dt.date(dt.strptime(y, '%d.%m.%Y'))),
+            ('ДатаКонца', lambda y: dt.date(dt.strptime(y, '%d.%m.%Y'))),
+            ('НачальныйОстаток', lambda y: float(y)),
+            ('ВсегоПоступило', lambda y: float(y)),
+            ('ВсегоСписано', lambda y: float(y)),
+            ('КонечныйОстаток', lambda y: float(y))
+        ]
+        for x in data_and_action:
+            try:
+                result['remain'][x[0]] = x[1](result['remain'][x[0]])

```

Comment:
Nicer if you check that the data exists in result['remain'] instead of catching AttributeError.